full-service/src/service/wallet.rs

```diff
@@ -122,6 +139,18 @@ pub enum JsonCommandRequest {
 #[serde(tag = "method", content = "result")]
 #[allow(non_camel_case_types)]
 pub enum JsonCommandResponse {
+    set_password {
+        success: bool,
+    },
+    unlock {
+        success: bool,
+    },
+    change_password {
+        success: bool,
+    },
+    is_locked {
+        is_locked: Option<bool>,
```

Comment:
Why is this optional?

Reply:
Perhaps I'm overloading the Option and this should be an enum - the meaning is:

None => Never been locked (no password set)
Some(true) => Is locked
Some(false) => Is unlocked

Reply:
Yeah, this should be an enum